docs/flashbots-protect/nonce-management.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Nonce Management
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Normally, a wallet can call `eth_getTransactionCount` to get the next nonce to use for a transaction.
+However, since transactions sent to Flashbots Protect are potentially sensitive, even exposing the incremented nonce can leak information about the user's activity.
+
+As such, Flashbots Protect requires that all requests to `eth_getTransactionCount` be signed by the user when querying the `"pending"` nonce.
+
+To query the `"pending"` nonce, requests must be signed with the user's private key. This is done by sending a JSON-RPC request to the Flashbots Protect RPC endpoint with the following parameters:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "method": "eth_getTransactionCount",
+  "params": [
+    "0xYOUR_ADDRESS",
+    "pending"
+  ],
+  "id": 1
+}
+```
+
+The request is signed and the signature is included in the `X-Flashbots-Signature` header.
+
+### Authentication
+
+To authenticate your request, Flashbots endpoints require you to sign the payload and include the signed payload in the `X-Flashbots-Signature` header of your request.
+
```

Comment:
We might want to (instead, or in addition) link directly to the authentication section in our API docs https://docs.flashbots.net/flashbots-auction/advanced/rpc-endpoint#authentication.

Reply:
I thought about that, but that section is relay-specific so includes some wording such as:

> Any valid ECDSA-secp256k1 key, like an arbitrary Ethereum key, can be used to sign the payload. The address associated with this key will be used by Flashbots to keep track of your [reputation](https://flashbots-docs-git-protect-ethgettransactioncount-flashbots.vercel.app/flashbots-auction/advanced/reputation) over time and provide user statistics. You can change the key you use at any time.

That conflicts with the usage here.